addons/product/models/product_pricelist.py

```diff
@@ -278,7 +276,6 @@ def _get_partner_pricelist_multi(self, partner_ids, company_id=None):
             # get fallback pricelist when no pricelist for a given country
             pl_fallback = (
                 Pricelist.search(pl_domain + [('country_group_ids', '=', False)], limit=1) or
-                Property._get('property_product_pricelist', 'res.partner') or
```

Comment:
This behavior should not be modified, but the field value should be empty if no pricelist exists in the db